clowdapp.yaml

```diff
@@ -28,11 +28,11 @@ objects:
         args: ["-c", "export FLASK_APP=manage.py && poetry run flask db upgrade && poetry run flask seed && poetry run gunicorn ros.api.main:app"]
         resources:
           requests:
-            cpu: ${CPU_REQUEST}
-            memory: ${MEMORY_REQUEST}
+            cpu: ${CPU_REQUEST_ROSRHEL}
```

Comment:
Are you going to create an MR on the app-interface side? You would need to update it in the [deploy.yml](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/insights/ros/deploy-clowder.yml) as well.

Reply:
Yes, those PRs are ready, I will share the same with you as soon as I merge these app side ones